### DISCUSSION TITLE
MHIG Maps will not load

### DISCUSSION THREAD

**Discussion by NAUDIS3** [Mr-MIBonk/M.I.B._More-Incredible-Bash#481] [Q&A]
I have been able to get my FEC legal. I am currently running MHIG_US_AU_K2019 and MIB successfully patched my unlimited maps to legal. I am using MIB 3.7.2. I found it when you posted it for someone else . 3.7.1 would not work for me. i have tried 4 different SD and 2 different microSD. Formatted using 3 different apps and unzipped with 2 different apps. I have tried using the 2024 maps and 2024-2025 maps. My unit will tell me NO valid update data found on the medium. I will drop logs later today.

**Comment by wanart105** [comment]
I’m experiencing the same issue.
